docs/designers-developers/developers/tutorials/format-api/1-register-format.md

```diff
@@ -0,0 +1,49 @@
+# Register a new format
+
+The first thing you're going to do in this tutorial is to register the new format that the plugin intends to apply. WordPress has the [`registerFormatType`](/packages/rich-text/README.md#registerFormatType) primitive to do so.
```

Comment:
Not a block for merge but I think it might be better to refer to this as a "function" which will be a more familiar term for less technical users.

Reply:
Done at 59b28818d94a376ebf2240b2ede2739c93d3ed3a

Reply:
Woot, you rock 🚀